src/parsing.jl

```diff
@@ -162,26 +162,70 @@ function parse_row!(rec::Records, row::Int, bytes, pos, len, options, eol_option
     return pos, code
 end
 
-function parse_row!(rec::TwoWindingTransformers, row::Int, bytes, pos, len, options, eol_options)
-    # Each `TwoWindingTransformers` is 4 lines of the file
-    # cols_per_line = (14, 3, 16, 2)
-    eol_cols = (14, 17, 33, 35)
-    ncols = nfields(rec)
-    @assert ncols == last(eol_cols)
+
+###
+### transformers
+###
+
+# Transformers data is a bit special, as records have 2 possible schemas
+# see https://github.com/nickrobinson251/PowerFlowData.jl/issues/17
+#
+# Each "two-winding transformer" is 4 lines with (14, 3, 16, 2) columns each, and
+# each "three-winding transformer" is 5 lines with (14, 11, 16, 16, 16) columns each.
+const T2_COLS = (14,  3, 16,  2)
+const T3_COLS = (14, 11, 16, 16, 16)
+const EOL_COLS = cumsum(T3_COLS)
+
+# To hold "three-winding" data we need `sum((14, 11, 16, 16, 16)) == 73` columns, and
+# column 14+3=17 and column 14+11+16+2=43 are "special" in that they may or may not be at
+# the end of a line.
+function parse_row!(rec::Transformers, row::Int, bytes, pos, len, options, eol_options)
+    ncols = fieldcount(Transformers)
+    @assert ncols == last(EOL_COLS)
+
     local code::Parsers.ReturnCode
-    for col in 1:ncols
+    col = 1
+    is_2w = false
+    while col < ncols
         eltyp = eltype(fieldtype(typeof(rec), col))
```

Comment:
if using `Vector{Union{T, Missing}}` (rather than `SentinelArray`), this may need to be `nonmissingtype(eltype(...))`